lib/src/template-context.ts

```diff
@@ -215,6 +215,12 @@ const makeTemplateContext = (): TemplateContext => {
 
 const originalPathParam = /:(\w+)/g;
 const getOriginalPathWithBrackets = (path: string) => path.replaceAll(originalPathParam, "{$1}");
+// Example full schema name is like: #/components/schemas/Category.
+// We only want to get the "Category".
+//
+// This is because when using `sortObjKeysFromArray`, the string array needs to be exactly the same
+// like the object keys. Otherwise, the object keys won't be re-ordered.
+const getPureSchemaNames = (fullSchemaNames: string[]) => fullSchemaNames.map((name) => name.split("/").at(-1)!);
```

Comment:
@astahmer So it seems like the issue was caused by something like this:

```ts
sortObjKeysFromArray({
  Store: '...',
  Country: '...'
}, [
  '#/components/schemas/Country',
  '#/components/schemas/Store',
])
```

Considering that `sortObjKeysFromArray` needs the string array to have exactly the same keys, it's now like this with the `getPureSchemaNames`:

```ts
sortObjKeysFromArray({
  Store: '...',
  Country: '...'
}, [
  'Country',
  'Store',
])
```

Let me know if you have better function name ideas; or other approaches that you think are better. Thanks!

Reply:
LGTM 👍  thanks